test/unittests/capi_test.cpp

```diff
@@ -1195,6 +1195,127 @@ TEST(capi, get_type)
     fizzy_free_module(module_imported_func);
 }
 
+TEST(capi, get_import_count)
+{
+    /* wat2wasm
+      (module)
+    */
+    const auto wasm_empty = from_hex("0061736d01000000");
+
+    const auto* module_empty = fizzy_parse(wasm_empty.data(), wasm_empty.size());
+    ASSERT_NE(module_empty, nullptr);
+
+    EXPECT_EQ(fizzy_get_import_count(module_empty), 0);
+    fizzy_free_module(module_empty);
+
+    /* wat2wasm
+      (func (import "m" "f") (result i32))
+      (global (import "m" "g") i32)
+      (table (import "m" "t") 0 anyfunc)
+      (memory (import "m" "m") 1)
+    */
+    const auto wasm = from_hex(
+        "0061736d010000000105016000017f021d04016d01660000016d0167037f00016d017401700000016d016d0200"
+        "01");
+
+    const auto* module = fizzy_parse(wasm.data(), wasm.size());
+    ASSERT_NE(module, nullptr);
+
+    EXPECT_EQ(fizzy_get_import_count(module), 4);
+    fizzy_free_module(module);
+}
+
+TEST(capi, get_import_description)
+{
+    /* wat2wasm
+      (func (import "m" "f1") (result i32))
+      (func (import "m" "f2") (param i64))
```

Comment:
Would it make any difference in coverage having a case with multiple params and a result at the same time? Don't want to overcomplicate tests either.

Reply:
I'll add it for completeness.